packages/core/src/icon/collections/core.ts

```diff
@@ -158,5 +162,7 @@ declare module '@cds/core/internal' {
     [vmBugIconName]: string;
     [vmBugInverseIconName]: string;
     [warningStandardIconName]: string;
+    [detailExpandIconName]: string;
+    [detailCollapseIconName]: string;
```

Comment:
new icons for the detail expand states that are used in the datagrid and action expand button